src/app/zap-templates/templates/app/attribute-size-src.zapt

```diff
@@ -86,7 +86,7 @@ uint16_t emberAfCopyList(ClusterId clusterId, EmberAfAttributeMetadata * am, boo
                 {{chipType}} * entry = reinterpret_cast<{{chipType}} *>(write ? src : dest);
                 {{#chip_attribute_list_entryTypes}}
                 {{#if (isString type)}}
-                chip::ByteSpan * {{name}}Span = &entry->{{name}}; // {{type}}
+                chip::ByteSpan * {{name}}Span = reinterpret_cast<chip::ByteSpan *>(&entry->{{name}}); // {{type}}
```

Comment:
why are we reinterpret-cast here? why is the entry name not of the correct type?

Reply:
That's what https://github.com/project-chip/connectedhomeip/issues/6801 was about.